### PR TITLE
Remove IML files and updated gitignore

### PR DESCRIPTION
**Pull request by adrianlshaw**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="Git" />
```
Comment (chorhatarahuduketuri) [inline .idea/vcs.xml]: Why is this removed?

Comment (adrianlshaw) [reply .idea/vcs.xml]: Shall we just remove this file altogether?

Comment (chorhatarahuduketuri) [reply .idea/vcs.xml]: Apparently the docs say we should have .idea in VCS, and since this is a file in .idea that describes VCS, I'm now for just keeping it in.

Comment (chorhatarahuduketuri) [reply .idea/vcs.xml]: Also, checking out this branch causes my IDE to no longer recognise the project.

Comment (adrianlshaw) [reply .idea/vcs.xml]: Still unsure what the values in this file should really be. The project made this change by itself and it seems simpler from my POV.

Comment (adrianlshaw) [reply .idea/vcs.xml]: Ah OK. Wonder which file is causing the issue.

Comment (chorhatarahuduketuri) [reply .idea/vcs.xml]: I think it's the removal of app.iml

Comment (chorhatarahuduketuri) [reply .idea/vcs.xml]: I'm gona reject this pull request until we decide what to add/remove from .gitignore. 

See under 'Directory-based format' here: https://www.jetbrains.com/help/idea/2017.1/about-projects.html

**Comment by chorhatarahuduketuri** [comment]
Reopening because I closed it prematurely.

File: app/app.iml [hunk 2]
```diff
@@ -1,110 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chorhatarahuduketuri) [inline app/app.iml]: Every attempt I've made to remove this file breaks the project in Android Studio. We need to keep it.

File: gc.iml [hunk 3]
```diff
@@ -1,19 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chorhatarahuduketuri) [inline gc.iml]: Every attempt I've made to remove this file breaks the project in Android Studio. We should keep it.
